src/Game/Jobs/RDM.ts

```diff
@@ -123,6 +134,21 @@ export class RDMState extends GameState {
 		this.registerRecurringEvents();
 	}
 
+	override jobSpecificAddDamageBuffCovers(node: ActionNode, skill: Skill<PlayerState>): void {
+		if (this.hasResourceAvailable(ResourceType.Embolden) && skill.aspect !== Aspect.Physical) {
+			node.addBuff(BuffType.Embolden);
+		}
+		if (this.hasResourceAvailable(ResourceType.Manafication)) {
```

Comment:
```suggestion
		if (this.hasResourceAvailable(ResourceType.Manafication) && skill.kind === "spell" || skill.kind === "weaponskill") {
```
manafic doesn't affect ogcds